app.js

```diff
@@ -188,6 +186,15 @@ app.post('/migrate', function (req, res) {
 
                         return jiraFetchIssues(project, firstIssue, lastIssue)
                             .then(function () {
+                                project.projects = Array.from(project.projects);
```

Comment:
Just wondering why we have to convert to Array here?  We use it as a Map so this may break things later?